oak_restricted_kernel/src/payload.rs

```diff
@@ -42,17 +42,25 @@ pub unsafe fn run_payload(payload: *const u8) -> ! {
         header.e_phnum as usize,
     );
 
-    for phdr in phdrs {
+    for phdr in phdrs.iter().filter(|&phdr| phdr.p_type == PT_LOAD) {
         let vaddr = VirtAddr::new(phdr.p_vaddr).align_down(Size2MiB::SIZE);
```

Comment:
pre-existing: Just in  case: Should Size2MiB be PAGE_SIZE?

Reply:
We don't have PAGE_SIZE constants here.